ert_shared/ensemble_evaluator/config.py

```diff
@@ -0,0 +1,36 @@
+import os
+import yaml
+import logging
+
+logger = logging.getLogger(__name__)
+CLIENT_URI = "client"
+DISPATCH_URI = "dispatch"
+DEFAULT_PORT = "8765"
+DEFAULT_HOST = "localhost"
+DEFAULT_URL = f"ws://{DEFAULT_HOST}:{DEFAULT_PORT}"
+CONFIG_FILE = "ee_config.yml"
+
+DEFAULT_EE_CONFIG = {
+    "host": DEFAULT_HOST,
+    "port": DEFAULT_PORT,
+    "url": DEFAULT_URL,
+    "client_url": f"{DEFAULT_URL}/{CLIENT_URI}",
+    "dispatch_url": f"{DEFAULT_URL}/{DISPATCH_URI}",
+}
+
+
+def load_config(config_path=CONFIG_FILE):
+    if not os.path.exists(config_path):
```

Comment:
Are we sure we want to be this forgiving? It seems like the code currently only uses the default, but there is no trace of an `ee_config.yml`. Hence, I guess `DEFAULT_EE_CONFIG` is used. Should we instead default `config_path=None` and then try to open `config_path` if it is not `None`? And fail if it does not exist?